lab-anthony/app/component/list/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+require('./list.scss');
+const angular = require('angular');
+const noteApp = angular.module('noteApp');
```

Comment:
I's generally considered best practice to: 
  * never store modules in variables like this (only use chaining)
  * only define one thing on a module in a given file, whether thats a directive or controller or whatever.